### PR TITLE
turn on pluto sync

### PR DESCRIPTION
**Pull request by akash1810**
TODO
- [x] send atom title in the message to Pluto
- [ ] have this flag as a server side config rather than a client-side POST body field
- [x] update IAM Role, granting Pluto permission to create temporary download links for simpler ingestion by videspine

### CODE REVIEW

**Comment by akash1810** [comment]
cc @fredex42

File: app/controllers/UploadController.scala [hunk 1]
```diff
@@ -78,7 +78,10 @@ class UploadController(override val authActions: HMACAuthActions, awsConfig: AWS
       projectId = atom.plutoData.flatMap(_.projectId),
       s3Key = CompleteUploadKey(awsConfig.userUploadFolder, id).toString,
       assetVersion = -1,
```
Comment (mbarton) [inline app/controllers/UploadController.scala]: I don't think we use this now?

Comment (fredex42) [reply app/controllers/UploadController.scala]: assetVersion, you mean?

Comment (mbarton) [reply app/controllers/UploadController.scala]: Yeah, I'm working on a different feature branch and I could see it being assigned but never read

File: app/controllers/UploadController.scala [hunk 2]
```diff
@@ -78,7 +78,10 @@ class UploadController(override val authActions: HMACAuthActions, awsConfig: AWS
...
-      atomId = atom.id
+      atomId = atom.id,
+      title = atom.title,
+      user = user.email,
+      posterImageUrl = atom.posterImage.flatMap(_.master).map(_.file)
```
Comment (akash1810) [inline app/controllers/UploadController.scala]: Why these new properties? As far as I can tell, we're not using them?

File: cloudformation/media-atom-maker.yaml [hunk 3]
```diff
@@ -199,6 +199,17 @@ Resources:
             Action:
             - kinesis:Get*
             - kinesis:DescribeStream
+      - PolicyName: PlutoIntegrationS3Policy
```
Comment (akash1810) [inline cloudformation/media-atom-maker.yaml]: can this be `PlutoIntegrationS3GetPolicy` to be more explicit?

File: app/controllers/UploadController.scala (hunk 2 above)
Comment (fredex42) [reply app/controllers/UploadController.scala]: i am using them at the pluto end, to make it easier to find masters if one does not make it through or the "launch detector" phase does not work.  They are what we are likely to want to search on.

File: cloudformation/media-atom-maker.yaml (hunk 3 above)
Comment (fredex42) [reply cloudformation/media-atom-maker.yaml]: sorry I don't understand?

Comment (fredex42) [reply cloudformation/media-atom-maker.yaml]: on re-reading I think i get it now :)

File: app/controllers/UploadController.scala (hunk 1 above)
Comment (fredex42) [reply app/controllers/UploadController.scala]: it's just there 'cos it was - never touched it. Can remove if you like or let it get removed in another update, up to you

Comment (mbarton) [reply app/controllers/UploadController.scala]: I'll remove it in my branch. No worries

**Comment by akash1810** [comment]
👍

**Comment by fredex42** [comment]
cool. I'll merge this and deploy tomorrow morning then?

**Comment by akash1810** [comment]
feel free to merge now, it'll go to CODE automatically

**Pull request merged**